packages/flame/lib/src/game/mixins/loadable.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:meta/meta.dart';
+
+mixin Loadable {
```

Comment:
nice abstraction! maybe a brief dartdoc on the mixin level for when it's used?

Reply:
Added!